test/Microsoft.ML.Benchmarks/TrainPredictionBench.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Running;
+using Microsoft.ML.Models;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Trainers;
+using Microsoft.ML.Transforms;
+
+namespace Microsoft.ML.Benchmarks
+{
+    public class TrainPredictionBench
```

Comment:
>TrainPredictionBench [](start = 17, length = 20)

Split benchmarks by task (regression, binary, etc). You already have classification metrics as a field. Perhaps have a base class and then subclasses for each task.

Reply:
changed the test to StochasticDualCoordinateAscentClassifierBench

Reply:
Should the file name be updated too?

---
In reply to: [187092340](https://github.com/dotnet/machinelearning/pull/62#discussion_r187092340) [](ancestors = 187092340)

Reply:
can you please update the file name to match the class name?

---
In reply to: [187243814](https://github.com/dotnet/machinelearning/pull/62#discussion_r187243814) [](ancestors = 187243814,187092340)